CharlieBackend.Api/Controllers/SecretariesController.cs

```diff
@@ -24,6 +24,15 @@ public SecretariesController(ISecretaryService secretaryService, IAccountService
             _accountService = accountService;
         }
 
+        /// <summary>
+        /// Gives secretary role to account and creates secretary entity
+        /// </summary>
+        /// <param name="accountId">Account id to approve</param>
+        /// <response code="200">Successful secretary entity create</response>
+        /// <response code="400">Account already assigned to secretary</response>
+        /// <response code="404">Acccount not found</response>
+        /// <response code="500">Can not assign account to secretary</response>
```

Comment:
500 error happens when we didn't handle some case and something went wrong on the server. We do not need to declare it in our contract.

Reply:
Ok